cmd/tapcli/assets.go

```diff
@@ -77,6 +78,11 @@ var mintAssetCommand = cli.Command{
 			Name:  assetSupplyName,
 			Usage: "the target supply of the minted asset",
 		},
+		cli.Uint64Flag{
+			Name: assetDecimalPlacesName,
+			Usage: "the number of decimal places that this asset " +
```

Comment:
I think this requires more info as well, can re-use most of what I suggested for the gRPC field description.